schema/changelog-3.10.xml

```diff
@@ -62,17 +62,43 @@
     </createTable>
 
     <addForeignKeyConstraint baseTableName="user_user" baseColumnNames="userid" constraintName="fk_user_user_userid" referencedTableName="users" referencedColumnNames="id" onDelete="CASCADE" />
-    <addForeignKeyConstraint baseTableName="user_user" baseColumnNames="manageduserid" constraintName="fk_user_user_manageduserid" referencedTableName="users" referencedColumnNames="id" onDelete="CASCADE" />
 
     <update tableName="users">
         <column name="devicelimit" valueNumeric="-1" />
         <where>devicelimit = 0</where>
     </update>
+    <dropDefaultValue tableName="users" columnName="devicelimit" />
```

Comment:
Unfortunately required. For MS SQL.
My bad, liquibase documentation confused me.